src/lazyload-image.directive.ts

```diff
@@ -1,44 +1,61 @@
 import 'rxjs/add/operator/let';
-import { Directive, ElementRef, Input, NgZone } from '@angular/core';
-import { getScrollListener } from './scroll-listener';
-import { lazyLoadImage } from './lazyload-image';
+import {Directive, ElementRef, Input, NgZone, SimpleChanges, OnChanges, OnDestroy} from '@angular/core';
+import {getScrollListener} from './scroll-listener';
+import {lazyLoadImage} from './lazyload-image';
+import {WindowService} from "./window.service";
```

Comment:
You are using spaces around the imported variable in your test file but not here. Any reasons?